src/components/form/errors/ApiErrorMsg.jsx

```diff
@@ -0,0 +1,41 @@
+
+import React from 'react';
+import { Message } from 'semantic-ui-react';
+import { PropTypes } from 'prop-types';
+
+/**
+ * Api error message component
+ * @param {Object} props contains {
+ * apiValidationErrors(Object): the validation errors from the api
+ * handleErrMsgDismiss(Function): a function to called to dismiss the msg
+ * displayErrMsg(Boolean): display the message when true
+ * }
+ * @returns {Custom} jsx or null
+ */
+const ApiErrorMsg = (props) => {
+  const { apiValidationErrors, handleErrMsgDismiss, displayErrMsg } = props;
+  if (apiValidationErrors.message
```

Comment:
Can you pls explain the logic in this function for more clarity. Thanks

Reply:
There is a jsDoc comment at the top, I've also added more comments to the code.